neon_audio/speech.py

```diff
@@ -26,6 +26,8 @@
 from ovos_utils.signal import check_for_signal
 from mycroft_bus_client import Message, MessageBusClient
 
+import librosa
```

Comment:
cant this be done without dragging a new dep?

Reply:
Not sure if there's another method, but it was moved here: https://github.com/NeonGeckoCom/neon-skill-utils/blob/a9fa0de452004b613ec866a8ab62b107d476e947/neon_utils/file_utils.py#L186 .. IIRC, there isn't any pure-python package for audio transcoding; they all seem to pull from `libsndfile` or `ffmpeg`